frontend/Makefile

```diff
@@ -1,13 +1,11 @@
-.PHONY: all test
+.PHONY: all test clean
 
+PROTO_DIR = src/frontend/proto
+TEST_DIR = src/frontend/cfg_generator
+VPATH = $(PROTO_DIR)
```

Comment:
what's this used for?

Reply:
It's so the Makefile can find the dependencies for the rule on line 12 (%_pb2.py: %.proto)